templates/base/head_navbar.tmpl

```diff
@@ -121,12 +121,13 @@
 				</div><!-- end content create new menu -->
 			</div><!-- end dropdown menu create new -->
 
-			<div class="ui dropdown jump item tw-mx-0 tw-pr-2" data-tooltip-content="{{ctx.Locale.Tr "user_profile_and_more"}}">
-				<span class="text tw-flex tw-items-center">
+			<div class="ui dropdown jump item" data-tooltip-content="{{ctx.Locale.Tr "user_profile_and_more"}}">
+				<span class="text">
 					{{ctx.AvatarUtils.Avatar .SignedUser 24 "tw-mr-1"}}
-					<span class="only-mobile tw-ml-2">{{.SignedUser.Name}}</span>
+					<span class="only-mobile">{{.SignedUser.Name}}</span>
 					<span class="not-mobile">{{svg "octicon-triangle-down"}}</span>
 				</span>
+				{{if .IsAdmin}}<span class="navbar-profile-admin">admin</span>{{end}}
```

Comment:
Should this perhaps be localized instead?

Reply:
I guess no need, if it gets localized, we can't control the length (width)

And I think every site admin should have basic English ability (otherwise, how to type commands ...)

Reply:
```suggestion
				{{/* do not localize it, here it needs the fixed length (width) to make UI comfortable */}}
				{{if .IsAdmin}}<span class="navbar-profile-admin">admin</span>{{end}}
```